1748736161421	Sync.LogManager	DEBUG	Flushing file log
1748736161422	Sync.LogManager	DEBUG	Log cleanup threshold time: 1747872161422
1748736161423	Sync.LogManager	DEBUG	Done deleting files.
1748739761421	Sync.Service	DEBUG	User-Agent: Firefox/139.0 (Linux x86_64) FxSync/1.141.0.20250522210034.desktop
1748739761421	Sync.Service	INFO	Starting sync at 2025-06-01 07:02:41 in browser session dVKA5ttehGyA
1748739761422	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1748739761425	Sync.Status	INFO	Resetting Status.
1748739761425	Sync.Status	DEBUG	Status.service: error.sync.failed_partial => success.status_ok
1748739761427	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1748739761428	FirefoxAccounts	DEBUG	getOAuthToken enter
1748739761428	FirefoxAccounts	TRACE	getCachedToken returning cached token
1748739761428	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1748739761428	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1748739761429	Sync.SyncAuthManager	INFO	Getting sync key
1748739761430	FirefoxAccounts	DEBUG	getOAuthToken enter
1748739761430	FirefoxAccounts	TRACE	getCachedToken returning cached token
1748739761430	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1748739761430	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1748739761430	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1748739761430	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1748739761431	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1748739761434	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1748739761434	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1748739761435	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1748739761436	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1748739761436	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1748739761436	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1748739761436	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1748739761436	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1748739761438	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1748739761438	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1748739761442	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/264067527/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1748739761442	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1748739761442	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1748739761442	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1748739761443	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1748739761443	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1748739761449	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
